web/app/components/editable-field.ts

```diff
@@ -1,64 +1,92 @@
-// @ts-nocheck
-// TODO: Type this file
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
 import { action } from "@ember/object";
 import { scheduleOnce } from "@ember/runloop";
+import { assert } from "@ember/debug";
 
 export const FOCUSABLE =
   'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])';
 
-export default class EditableField extends Component {
-  @tracked editing = false;
-  @tracked element = null;
```

Comment:
`element` is a protected propertyName according to Glint/TS so I changed it to `el`